Customer/Method/index.ts

```diff
@@ -19,6 +20,20 @@ export namespace Method {
 		return is(result) ? result : undefined
 	}
 	export type Card = MethodCard
+	export async function toPayment(method: MethodCard | any): Promise<Payment.Card.Creatable | undefined> {
+		let result: Payment.Card.Creatable | undefined 
```

Comment:
```suggestion
		let result: Payment.Card.Creatable | undefined
```
Just to fix the warning.